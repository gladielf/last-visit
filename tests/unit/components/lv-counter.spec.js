import { shallowMount } from '@vue/test-utils';
import LvCounter from '@/components/lv-counter.vue';

describe('LvCounter - Snapshot', () => {
    it('Snapshot', () => {
        const wrapper = shallowMount(LvCounter);
        expect(wrapper.element).toMatchSnapshot();
    });
});
