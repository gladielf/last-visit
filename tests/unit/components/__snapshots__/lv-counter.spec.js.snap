// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LvCounter - Snapshot Snapshot 1`] = `
<div
  class="lv-counter"
>
  <p
    class="lv-counter__digit"
  >
    
  </p>
  <p
    class="lv-counter__label"
  >
    
  </p>
</div>
`;
